modules/upgrading-the-odh-operator.adoc

```diff
@@ -0,0 +1,28 @@
+:_module-type: PROCEDURE
+
+[id="upgrading-the-odh-operator_{context}"]
+= Upgrading the Open Data Hub Operator
+
+.Prerequisites
+* You have installed version 1 of the Open Data Hub Operator.
+* You are using OpenShift Container Platform 4.11 or later.
+* Your OpenShift cluster has a minimum of 16 CPUs and 32GB of memory across all OpenShift worker nodes.
+* You can log in as a user with `cluster-admin` privileges.
+
+.Procedure
+. Log in to the OpenShift Container Platform as a user with `cluster-admin` privileges.
+. Select *Operators* -> *Installed Operators*, and then click the 1.X version of the *Open Data Hub Operator*.
+. Click the *Subscription* tab.
+. In the *Update channel* section, click the pencil next to *rolling*.
+. In the *Change Subscription update channel* box, select *fast*, and then click *Save*.
```

Comment:
Suggest calling this a dialog, rather than a box.

Reply:
Also, format the name of the channel `fast` as a literal value by enclosing in backticks.

Reply:
Changed to "window" per https://docs.engineering.redhat.com/display/CCSENABLE/UI+elements

Reply:
Hmmm, I would actually question that Red Hat guidance - it's a little incomplete. This is a window that appears on top of the main page. So, I would still call it a _dialog_, as described in the IBM Style Guide: https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#dialog.